src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -676,6 +710,40 @@ def readImage(self):
         else:
             raise Exception('Unknown error '+ hex(receivedPacketPayload[0]))
 
+    def readImageWithoutLighting(self):
```

Comment:
It would be nicer if `readImage()` would get an optional (boolean) parameter that determines to turn LED on or off. As I see `readImageWithoutLighting()` is a copy of `readImage()`

Reply:
Yeah, the only difference is if it sends `0x01` or `0x52` as the command. I'll get that changed.